conference.js

```diff
@@ -2343,6 +2345,7 @@ export default {
             'device count: ' + videoDeviceCount);
 
         APP.store.dispatch(setVideoAvailable(available));
+        APP.API.notifyVideoAvailabilityChanged(available);
```

Comment:
I'm wondering these API calls are better to do in redux/actions/middleware. However, it would make the logic more difficult to detect while conference.js is still in heavy use. Is it tricky to move these calls out of conference and into the react/redux app world?